nixos/modules/services/misc/nitter.nix

```diff
@@ -155,6 +160,22 @@ in
           description = lib.mdDoc "Use base64 encoding for proxied media URLs.";
         };
 
+        enableRSS = mkEnableOption (lib.mdDoc "Set this to false to disable RSS feeds.");
```

Comment:
`mkEnableOption` defaults to `false`.

Use `mkEnableOption "[...]" // { default = true; }` if it should default to `true`.